.github/actions/artifacts/do-upload.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
please do not write new shell scripts -- this should be written in a language which can have proper type checking and tests.  python is sentry's language of choice for such things

Reply:
Fixed this if you want to take another look!